src/types.h

```diff
@@ -154,6 +155,8 @@ class SizedType {
   AddrSpace as_ = AddrSpace::none;
   bool is_signed_ = false;
   bool ctx_ = false; // Is bpf program context
+  std::optional<std::string> btf_type_tag_ = std::nullopt; // Only populated for
```

Comment:
Is optional necessary? std::string::empty() is unambiguous, right?

Reply:
Also, looking at https://www.kernel.org/doc/html/next/bpf/btf.html#btf-kind-type-tag, seems like multiple type tags are supported. so maybe this should even be a vector

Reply:
Oh good catch. Let me look into supporting that. I guess if any in that list are not "rcu" we need to reject.